config/config.json

```diff
@@ -2,7 +2,6 @@
   "environment_variables": {
       "env_package_root" : "CC_PACKAGE_ROOT",
       "env_verbose_name" : "CC_VERBOSE_LEVEL",
```

Comment:
Is this used? I think commands moved to use the `--verbose` option. @gyorb ?

Reply:
Yes, `--verbose` should be used right now to set log levels.